### PR TITLE
Update: Add copy to clipboard for about brave page (fixes #5790)

### LINKED ISSUE #5790: about:brave page should have "copy to clipboard" button
I like the recent change to version info. It looks cleaner than presenting a dialog. One downside though is that selecting and copying the info gives me this:

Brave	0.12.10
Muon	1.4.26
libchromiumcontent	53.0.2785.143
V8	5.3.332.47
Node.js	6.5.0
Update Channel	dev
os.platform	linux
os.release	4.8.0-27-generic
os.arch	x64

It might be better to add a "copy to clipboard" button which provides something slightly easier to scan when pasted into issue tickets:

Brave: 0.12.10
Muon: 1.4.26
libchromiumcontent: 53.0.2785.143
V8: 5.3.332.47
Node.js: 6.5.0
Update Channel: dev
os.platform: linux
os.release: 4.8.0-27-generic
os.arch: x64

That's basically the text content format of the old dialog.

### PR DESCRIPTION
**Pull request by gyandeeps**
- [x] Submitted a [ticket](https://github.com/brave/browser-laptop/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [x] Added/updated tests for this change (for new code or code which already has tests).
- [x] Ran `git rebase -i` to squash commits (if needed).

Test Plan:

1. Open Brave
1. Go to Help > About
1. Click the copy to clipboard icon
1. try to paste the copied data into a doc

This is my first PR. I wasn't sure what sort test needs to be written for copy to clipboard functionality.

#### UI

 Same as mentioned in the issue https://github.com/brave/browser-laptop/issues/5790#issuecomment-265839135

#### Copied data to clipboard:

```
Brave: 0.12.13 
Muon: 1.4.28 
libchromiumcontent: 53.0.2785.143 
V8: 5.3.332.47 
Node.js: 6.5.0 
Update Channel: dev 
os.platform: win32 
os.release: 10.0.14393 
os.arch: x64
```

### CODE REVIEW

File: less/about/siteDetails.less [hunk 1]
```diff
@@ -82,6 +82,9 @@ body {
       font-size: 16px;
       margin-bottom: 12px;
       padding-left: @aboutPageSectionPadding;
+      width: 400px;
+      display: flex;
+      justify-content: space-between;
```
Comment (darkdh) [inline less/about/siteDetails.less]: I think maybe we should move these three styles to parent selector of `.sectionTitle`
so the original sectionTitle style will remain the same.
screen shot 2016-12-09 at 13 25 49

Comment (darkdh) [inline less/about/siteDetails.less]: And if we do so, `width: 424px` would be aligned

Comment (luixxiul) [inline less/about/siteDetails.less]: adding `margin-right` to `.aboutAbout .fa-clipboard` would work too.

Comment (darkdh) [inline less/about/siteDetails.less]: Sorry, I didn't put the icon aligned. What I meant is 
```
<div className='title'>
  <span className='sectionTitle' data-l10n-id='versionInformation' />
   <span className='fa fa-clipboard' title='Copy password to clipboard' onClick={this.onCopy} />
 </div>
```
We wanna keep the original attribute for version information like orange color.
So we can put the attributes of `display` and `justify-content` into `.title`
And as @luixxiul suggest, we can add `margin-right` to `.fa-clipboard`

Comment (gyandeeps) [inline less/about/siteDetails.less]: ah i see now. Sorry abt that I dint realize we lost the color of the title. Will make the change tonight.

**Comment by gyandeeps** [comment]
@darkdh I made all the changes mentioned in the comments. Let me know if anything else is needed.

**Comment by darkdh** [comment]
@gyandeeps , great job, thanks!

File: less/about/siteDetails.less [hunk 2]
```diff
@@ -78,10 +78,16 @@ body {
     display: block;
     clear: both;
 
+    .title {
+      width: 400px;
+      display: flex;
+      justify-content: space-between;
+      margin-left: @aboutPageSectionPadding;
```
Comment (luixxiul) [inline less/about/siteDetails.less]: Should this not be margin-right?

Comment (darkdh) [inline less/about/siteDetails.less]: screen shot 2016-12-10 at 14 17 38

That is the same margin-left aligned with the sortable table below

**Comment by bbondy** [comment]
Sorry I had to revert this here: https://github.com/brave/browser-laptop/pull/6184
Because of https://github.com/brave/browser-laptop/issues/6183

**Comment by bbondy** [comment]
Please re-submit a PR once fixed. 

Just wanted to also mention I don't think we should be special casing this about page's section title for consistency reasons.

**Comment by luixxiul** [comment]
I will work on it with cherry-pick

**Pull request merged**